.travis.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       script:
         - cd conode
         - make docker
+        - docker run --rm -t -p 7770-7777:7770-7777 --name conode c4dt/conode:dev &
+        - make conode
+        - build/bcadmin latest --bc bc*cfg
+        - docker rm -f conode
```

Comment:
Why? It's the end of this run

Reply:
No cleaning up?

Reply:
Yup, no need to clean, as the server is taken down by travis anyway